x-pack/plugins/siem/public/components/flyout/header/index.tsx

```diff
@@ -32,9 +35,8 @@ interface OwnProps {
 interface StateReduxProps {
   description?: string;
   getNotesByIds: (noteIds: string[]) => Note[];
-  history?: History[];
   isFavorite?: boolean;
-  isLive?: boolean;
+  isLock?: boolean;
```

Comment:
consider `isLocked` (present tense) or `isDatepickerLocked`, which includes the subject of the lock